rubicon/objc/objc.py

```diff
@@ -1696,3 +1726,62 @@ def __call__(self, *args):
 class ObjCBlockInstance(ObjCInstance):
     def __call__(self, *args):
         return self.block(*args)
+
+
+_NSConcreteGlobalBlock = ObjCClass("__NSGlobalBlock__")
```

Comment:
The `__NSGlobalBlock__` class isn't actually documented anywhere. `_NSConcreteGlobalBlock` is [documented](https://clang.llvm.org/docs/Block-ABI-Apple.html#high-level), but not required to be an Objective-C class (you're only supposed to use its address).

So the officially supported way to use this would be:

```python
_NSConcreteGlobalBlock = (c_void_p * 32).in_dll(c, "_NSConcreteGlobalBlock")
# ... when setting the isa ...
self.literal.isa = addressof(_NSConcreteGlobalBlock)
```

(`_NSConcreteGlobalBlock` is declared as a `void *[32]` in [`Block.h`](https://opensource.apple.com/source/libclosure/libclosure-67/Block.h.auto.html).)